src/app/dialogs/download-historic-data-dialog/download-historic-data-dialog.component.ts

```diff
@@ -11,7 +11,7 @@ import {environment} from '../../../environments/environment';
 export class DownloadHistoricDataDialogComponent implements OnInit {
   sortedVaults: string[];
   includeInactive = false;
-  apiEndpoint= "";
+  apiEndpoint = '';
```

Comment:
is this expected?

Reply:
Yes. Our ESLint configuration/guidelines use single quotes and it marks this as a listing error.